dotnet-efcore/EntityFrameworkCoreVsDapper/Tests/PersonRepositoryDapperUnitTests.cs

```diff
@@ -8,27 +8,11 @@ public class PersonRepositoryDapperUnitTests
         private PersonsRepositoryDapper repository = new PersonsRepositoryDapper();
 
         [Fact]
```

Comment:
You could make this a Theory and just pass in the 100, 1000, and 10000 values. It will shorten the code and make it more maintainable.